src/main/java/com/mjbmjb/cf/taskmaster/taskmaster/controller/TaskController.java

```diff
@@ -61,11 +73,29 @@ public RedirectView updateStatus(@RequestParam String id) {
 
     }
 
+    @CrossOrigin
+    @PostMapping("/tasks/{id}/images")
+    public RedirectView uploadImage(
+            @RequestParam String id,
+            @RequestPart(value = "file") MultipartFile file
+    ) {
+        Optional<TaskMaster> current = taskMasterRepository.findById(id);
+        TaskMaster currentTaskMaster = current.get();
+        String pic = this.s3Client.uploadFile(file);
+
+        currentTaskMaster.setPicture(pic);
+        taskMasterRepository.save(currentTaskMaster);
+
+        return new RedirectView("http://localhost:3000");
```

Comment:
This will only work when you're testing it locally!!! Be sure to update this to the deployed link!